README.md

```diff
@@ -1,46 +1,155 @@
+    <T> Observable.Transformer<T, T> applySchedulers();
+}
+```
+
+```
+public final class ProductionSchedulersTransformer implements SchedulersTransformer {
+
+    private final Observable.Transformer schedulersTransformer;
+
+    public ProductionSchedulersTransformer() {
+        schedulersTransformer = observable -> ((Observable) observable)
+                .subscribeOn(Schedulers.computation())
+                .observeOn(AndroidSchedulers.mainThread());
+    }
+
+    @Override
+    public <T> Observable.Transformer<T, T> applySchedulers() {
+        return (Observable.Transformer<T, T>) schedulersTransformer;
+    }
+}
+```
+
+```
+public final class RxEspressoTransformer implements SchedulersTransformer {
 
-There *is* **one big exception**. All observables must complete before Espresso will continue.  (I'm still trying to figure out a solution to this).  Below are some of the common use cases that will cause problems:
+    private final Observable.Transformer schedulersTransformer;
 
-* You cannot use RxJava in an "event bus" style where you have an Observable that forever remains "hot" to serve up events to interested subscribers.  If you are using RxJava in this manner and want to use Espresso, you will need to migrate to another event bus option.  My preference is using [EventBus](https://github.com/greenrobot/EventBus)
-* You will also run into issues if you have Observables running outside the scope of the UI being tested, such as in an Android Service.  The specifics of your usage will determine the level of impact.  Possibly disabling the services when being tested by Espresso could solve the issue.
+    public RxEspressoTransformer() {
+        schedulersTransformer = observable -> ((Observable) observable)
+                .subscribeOn(Schedulers.computation())
+                .observeOn(AndroidSchedulers.mainThread())
+                .doOnSubscribe(() -> RxEspresso.increment())
+                .doAfterTerminate(() -> RxEspresso.decrement());
+    }
+
+    @Override
+    public <T> Observable.Transformer<T, T> applySchedulers() {
+        return (Observable.Transformer<T, T>) schedulersTransformer;
+    }
+}
+```
 
-## Tips
-If you do have a situtation where the tests aren't running as expected due to a never-finishing Observable, you can:
+Then in code we call our injected instance of `SchedulersTransformer`:
 
-1. Enable logging by calling `RxEspresso.setLogLevel(LogLevel.DEBUG)` somewhere in your test setup, and filter you logcat based on the TAG: `RxIdlingResource`.  By looking at the Object reference IDs, you can identify any that have started but not completed.
+```
+@Inject SchedulersTransformer schedulersTransformer;
 
-2. Once you've identified one that never completes, you can enable logging with stack traces to see where this Observer subscription originates from by calling `RxEspresso.setLogLevel(LogLevel.VERBOSE)`
+dataStore.getData()
+     .compose(schedulersTransformer.applySchedulers())
```

Comment:
I would like to see naming that makes this more clear somehow.  Maybe something like `.compose(fragmentTransformer.applyUiSchedulers())`?  Something that signifies these are for use in the UI layer (Fragments).  I'm not sure exactly but it seems like it could be clarified.